src/tiled/objectselectiontool.cpp

```diff
@@ -559,17 +558,16 @@ void ObjectSelectionTool::mousePressed(QGraphicsSceneMouseEvent *event)
                 auto selection = mapScene()->selectedObjectItems();
                 if (event->modifiers() & (Qt::ShiftModifier | Qt::ControlModifier)) {
                     if (selection.contains(objectToBeSelected))
-                            selection.remove(objectToBeSelected);
-                        else
-                            selection.insert(objectToBeSelected);
+                        selection.remove(objectToBeSelected);
+                    else
+                        selection.insert(objectToBeSelected);
                 } else {
                     selection.clear();
                     selection.insert(objectToBeSelected);
                 }
                 mapScene()->setSelectedObjectItems(selection);
             }
-        }
-        else
+        } else
```

Comment:
Coding style: if some if/else chain has brackets somewhere, it should have them everywhere. Please add `{` here.